### PR TITLE
Implement support for hosts with multiple values and ranges

### PR DESCRIPTION
**Pull request by cyrixsimon**

### CODE REVIEW

**Comment by cyrixsimon** [comment]
Resolves #14

File: websocket.go [hunk 1]
```diff
@@ -8,31 +8,77 @@ import (
...
+		if c.config.Encryption {
+			scheme = "wss"
+		}
+
+		u := url.URL{
+			Scheme: scheme,
+			Host:   uri,
+		}
+		var ws *websocket.Conn
+		log.Printf("Connect to %s", u.String())
```
Comment (Nightapes) [inline websocket.go:73]: Could you remove the` log.Printf`, at the moment we print only errors

**Review by Nightapes — CHANGES_REQUESTED** [review]
@cyrixsimon Thank you for the PR!

Could you resolve the conflicts in `websocket.go`

File: websocket.go [hunk 2]
```diff
@@ -8,31 +8,77 @@ import (
...
+	rand.Shuffle(len(hosts), func(i, j int) {
+		hosts[i], hosts[j] = hosts[j], hosts[i]
+	})
+	for _, host := range hosts {
+		uri := fmt.Sprintf("%s:%s", host, c.config.Port)
+
+		scheme := "ws"
+		if c.config.Encryption {
+			scheme = "wss"
+		}
```
Comment (AnastasiiaSergienko) [inline websocket.go:66]: Could you please extract a function here? "getPrefix" to something similar

**Review by AnastasiiaSergienko — COMMENTED** [review]
Thanks for your contribution! I left a small suggestion :)

**Comment by cyrixsimon** [comment]
What is going on with the linter here?

**Comment by cyrixsimon** [comment]
Oh this was my fault - i have resolved the merge conflict incorrectly. Now all checks passed and I have implemented all your suggestions.

**Review by AnastasiiaSergienko — APPROVED** [review]

**Review by Nightapes — APPROVED** [review]

**Pull request merged**